crates/aranya-client/tests/tests.rs

```diff
@@ -264,11 +268,11 @@ impl UserCtx {
     }
 
     async fn aranya_local_addr(&self) -> Result<SocketAddr> {
-        Ok(self.client.aranya_local_addr().await?)
+        Ok(self.client.local_addr().await?)
```

Comment:
Since we're going to use `client.afc` and `client.aqc` to differentiate AFC from AQC, we might as well use `client.sync` to separate out the sync related methods.
```suggestion
        Ok(self.client.sync().local_addr().await?)
```